1.Exploratory_Data_Analysis/all_anova_call_local.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+conda activate Interstellar_python
+
+jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/ notebooks/*.ipynb
+
+CELL_TYPE="SHSY5Y"
```

Comment:
Do you also run this with PBMCs? You might also consider looping over them like you do in line 18.